### ISSUE TITLE
Assumes UTC

### ISSUE THREAD

**Issue by brianpos** [ewoutkramer/fhir-net-api#731]
If the datetime is partial, i.e. has not time component, I think it is incorrect that we should be messing with the value, as there has been no mention of the time, and hence nothing to convert.
https://github.com/ewoutkramer/fhir-net-api/blob/5a58d95812e040e94a73a6298e6fc2718fcd8ad0/src/Hl7.Fhir.Core/Model/FhirDateTime.cs#L100

**Comment by ewoutkramer** [comment]
Yes, to be consistent with #724, we should also make this a non-optional.  In practice, we should split this method up in two (one with, one without the parameter) and [obsolete] the second.

**Comment by brianpos** [comment]
I think the compiler error in this case is fine. No need for the obsolete. Dont expect it's used that much, and a simple change.
